src/pages/debrief/components/vehicle-checks-card/vehicle-checks-card.html

```diff
@@ -16,7 +16,13 @@ <h4 class="fault-heading">{{ 'debrief.vehicleChecks' | translate }}</h4>
           </ion-col>
           <ion-col col-3 no-padding></ion-col>
           <ion-col no-padding>
-            <div class="counter-label">{{question.code}} - {{ 'debrief.showMeTellMeQuestions.' + this.category + '.' + question.code | translate }}</div>
+            <div class="counter-label" [ngSwitch]="this.category">
+              <span *ngSwitchCase="'F'">{{question.code}} - {{ 'debrief.showMeTellMeQuestions.FGHK.' + question.code | translate }}</span>
```

Comment:
Could maybe use ngIf isHomeTest() and have one span instead of 4 duplicates

Reply:
Sure, that's fair.